### ISSUE TITLE
number of roundtrips

### ISSUE THREAD

**Issue by SalomeRonja** [rinikerlab/reeds#57]
As we discussed recently, the final sopt analysis summary plot is not ideal at the moment, namely the "Total number of roundtrips" plot. The simulation time of the sopt runs increases with each iteration (500ps, 1000ps, 1500ps, 1500ps, ...) and this is not taken into account in the total number of roundtrips plot.

I had a look at the code today, and what the plot actually shows is not the total number of roundtrips, but the total number of replicas which perform at least one roundtrip. As I see it there are two options:

- leave the plot as it is now, but change the title to something like "number of replicas performing at least one roundtrip"
- change the plot so it shows the total number of roundtrips per ns

@SchroederB @candidechamp what do you think? If you let me know which solution you prefer (or you have a different idea for a metric that would be interesting for the summary plot), I can make the corresponding alterations to the code.

![benzene_set_watsopt_sopt_analysis](https://user-images.githubusercontent.com/13115540/136007808-0bc316e9-68d3-4f2a-8441-e78142e0a227.png)

**Comment by candidechamp** [comment]
Thanks for looking at this Salome. 

I suggest plotting the total #roundtrips per nanosecond. 
One could in principle also look at the total #roundtrips per nanosecond per #replicas. 

My second suggestion is because the plot would be the same if we observe 25 round-trips in 1ns whether we are running RE with 25 or 50 replicas. Where in one case, this suggests all replicas perform a round trip~ once, whereas in the other case only half do.

**Comment by SchroederB** [comment]
I agree with candide,

For the manuscript I used the first suggestion:
![image](https://user-images.githubusercontent.com/12428005/136012490-bad31157-f76d-4b40-a5fd-a5a6e0be7bd7.png)

**Comment by SchroederB** [comment]
I agree with candide,

For the manuscript I used the first suggestion:
![image](https://user-images.githubusercontent.com/12428005/136012603-0b4da289-fa1c-47ba-99b5-b497d901ec0b.png)

**Comment by SalomeRonja** [comment]
sounds good! Just to be sure we're all on the same page, what we want is to sum up the total number of roundtrips for all replicas, and then divide by the number of replicas and the simulation time, right?

**Comment by candidechamp** [comment]
Yes to me what you describe is really what I find most informative. Then we'd get number that are ~ 1 roundtrip / ns / replica (this is around what I get for ligands in water and in complex)

I would even add "error bars" to the plot to indicate how much variation there are among the different replicas. (for example taking the standard deviation of the #roundtrip/ns (of each replica))

**Comment by SalomeRonja** [comment]
I implemented the changes and started the pull request #58 

The new plot looks as follows:

![benzene_set_watsopt_sopt_analysis](https://user-images.githubusercontent.com/13115540/136052126-24053737-fdc4-45ef-acb2-369738a19572.png)

@candidechamp feel free to add error bars as well :)

**Comment by candidechamp** [comment]
@SalomeRonja I added some error bars (many months later haha) and also changed the Replica placement subplot in #65 (so it shows the s value on a log scaled x-axis and not -log(s)). Also the plot of the top right calculated the roundtrip time by assuming 20 steps would be done between exchanges, and I changed it such that the user could give the number they used as input.  

I guess we can close this issue now?
